modular_skyrat/modules/ashwalkers/code/buildings/railroad.dm

```diff
@@ -119,7 +119,14 @@
 		if(GetComponent(/datum/component/simple_farm) || !use_item.use(10))
 			return ..()
 		AddComponent(/datum/component/simple_farm, TRUE, TRUE, list(0, 16))
+		RemoveElement(/datum/element/ridable)
 		return
+
+	if(attacking_item.tool_behaviour == TOOL_SHOVEL)
+		var/datum/component/remove_component = GetComponent(/datum/component/simple_farm)
+		if(remove_component)
+			qdel(remove_component)
```

Comment:
Why did you RemoveElement in one spot, and just `qdel`'d it in the other? That's probably going to create a hard-del, no?

Reply:
I'm an idiot never mind